### DISCUSSION TITLE
Automatic dimming according to sunrise and sundown

### DISCUSSION THREAD

**Discussion by unentschieden85** [xanderfrangos/twinkle-tray#601] [Ideas]
The timing feature is nice, but as sunrise and sunset change during the year, it really is only perfect for a couple of weeks and needs constant changing during the year. It would be much better, if brightness (and contrast btw!) could be automatically adjusted according to the location and its sunrise and sunset.

**Comment by amid525** [comment]
Очень не хватает этого!
